pkg/operator/ceph/pool/controller.go

```diff
@@ -351,8 +351,11 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 				logger.Debug("pool monitoring go routine already running!")
 			} else {
 				r.blockPoolContexts[blockPoolChannelKey].started = true
-				// Run the goroutine to update the mirroring status
-				go checker.CheckMirroring(r.blockPoolContexts[blockPoolChannelKey].internalCtx)
+				// Run the goroutine to update the mirroring status and skip when blockpool mirroing mode in init-only as radosnamespace mirroring is the right place to check
+				// mirroring status when blockpool mirroring mode is init-only.
+				if cephBlockPool.Spec.Mirroring.Mode != "init-only" {
```

Comment:
Do we need to move inside if check `r.blockPoolContexts[blockPoolChannelKey].started = true`  and also see if we need change in below disable else block as well

Reply:
moved that inside if and no change is required in below else code as it only resetting the status to nil.